packages/lib/src/toolbar/floating/FloatingControl.tsx

```diff
@@ -0,0 +1,39 @@
+import type { ReactNode } from 'react';
+import { useContext, useEffect, useLayoutEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+
+import { AxisSystemContext } from '../../vis/shared/AxisSystemContext';
+
+interface Props {
+  children?: ReactNode;
+}
+
+function FloatingControl(props: Props) {
+  const { children } = props;
+  const { floatingToolbar } = useContext(AxisSystemContext);
+
+  // Container `div` for ReactDOM to render the control into, appended to floating toolbar
+  const [controlContainer] = useState(() => {
+    const el = document.createElement('div');
+    el.style.cssText = 'pointer-events: auto;'; // toolbar uses `Html` which disables `pointer-events`
```

Comment:
Remind me why `Html` needs to disable `pointer-events` ?

Reply:
Because it adds a `div` on top of the whole canvas. `Html` does a lot more than needed in some cases, so it may need to be split at some point.